docs/sql-data-sources-jdbc.md

```diff
@@ -275,11 +275,22 @@ logging into the data sources.
     </td>
     <td>read/write</td>
   </tr>  
+
+  <tr>
+   <td><code>connectionProvider</code></td>
+   <td>(none)</td>
+   <td>
+     The short name of the JDBC connection provider to use to connect to this URL, e.g. <code>db2</code>, <code>mssql</code>.
```

Comment:
What does `short` means here? `The name of a JDBC connection provider to use ...`?